content/components/accordion/rationale/usage.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: component/rationale
+heading: Using accordions
+---
+
+We recommend that accordions be used sparingly.  While they can be appropriate for organising small, specific sections of content, accordions are not a suitable replacement for well formattted plain text. If a user requires all, or most of the informmation on the page it should be visible, and not hidden inside an accordion. 
+
+[Things to consider](https://www.nngroup.com/articles/accordions-complex-content/) when using accordions are, 
+
+**Hiding content makes it harder for a user to scan a webpage.**  If content is hidden inside an accordion it can be harder for a user to scan a whole web page for content relevent to them and also makes it hard for a user to use `Control + F` to search for certain keywords. 
+
+**Accordions increase cognitive load.**  Forcing a user to click on each accordion in order to receive the full text fragments their user experience, causing them to switch focus between accordions in order to gather all the information. It is also possible that with hidden content, a user might ignore, or not see imortant information.  
+
+**Printing.** Accordions can be difficult to print correctly and may require users to print out several sinppets of content if not optimised correctly.
```

Comment:
Nice @KiriHoy,

We can get rid of the last paragraph on printing, since the DS accordion comes in built with print styles